src/main/scala/org/broadinstitute/dsde/rawls/workspace/WorkspaceService.scala

```diff
@@ -254,7 +254,7 @@ class WorkspaceService(userInfo: UserInfo, dataSource: DataSource, containerDAO:
     }
 
     // collect the result
-    eventualResponses map (r => RequestComplete(r.collect { case Some(x) => x }))
+    eventualResponses map (r => RequestComplete(StatusCodes.OK, r.collect { case Some(x) => x }))
```

Comment:
Actually, that isn't quite true, is it.  There are a few 2XX responses of the shape RequestComplete[StatusCode].  Ideally the status code should be NoContent for all of those, but I know for a fact that it isn't.